specs/mediawiki/v1/content.yaml

```diff
@@ -18,28 +18,33 @@ paths:
     get:
       tags:
         - Page content
-      description: List pages
-      x-backend-request:
-        uri: /{domain}/sys/page_revisions/page/
+      description: List page properties / page content sub-apis
 
-  /{module:page}/{title}:
+  /{module:page}/title/:
     get:
       tags:
         - Page content
-      description: Redirect to the latest HTML for this page
-      # TODO
-      # x-backend-request:
-      #   uri: /{domain}/sys/page_revisions/page/
+      description: List all known pages
+      x-backend-request:
+        uri: /{domain}/sys/page_revisions/page/
 
-  /{module:page}/{title}/:
+  /{module:page}/title/{title}/:
     get:
       tags:
         - Page content
-      description: List available content types for a page
+      description: List page meta-data
       produces:
         - application/json
 
-  /{module:page}/{title}/html:
+  /{module:page}/html/:
+    get:
+      tags:
+        - Page content
+      description: List titles for which the HTML property is available
+      x-backend-request:
+        uri: /{domain}/sys/page_revisions/page/
+
+  /{module:page}/html/{title}:
```

Comment:
Should we directly return the latest result here & send no-cache headers or explicitly purge this end point? Redirecting would only add latency once we are in restbase. Pros would be a) teaching users to actually use the revision URI, and b) a smaller cache entry if we cache / purge the redirect response.